package.json

```diff
@@ -33,7 +33,7 @@
     "handlebars": "^4.0.5",
     "mkdirp": "^0.5.1",
     "moment": "^2.14.1",
-    "typescript": "^2.1.0-dev.20160910",
+    "typescript": "^2.0.3",
```

Comment:
Here are my observations:
* typescript newer than 2.1.0-dev.20160910 at some time changes the formatOptions from uppercase to lowercase which breaks with certrain version combination of typescript-formatter and `unformats` all the source files
* typescript newer than 2.1.0-dev.20160910 introduced a bug in jsdoc parsing (https://github.com/Microsoft/TypeScript/issues/10987) with multiline comments which makes your multiline test break